### PR TITLE
Add DAWA (Denmarks Address Web API) extension

### PR DESCRIPTION
**Pull request by simonj**
## Description

This extension let's the user that works with Danish addresses search for unique address id's by using the https://dawadocs.dataforsyningen.dk/ API (Denmarks Addresses Web API)

All internet providers in Denmark use this API and a lot of other big companies use this API to get information about Danish addresses.

You can use this address ID to test with: **0a3f5081-c522-32b8-e044-0003ba298018**

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/dawa/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "denmarks-address-web-api",
+  "title": "Search Through The Danish Address Web API",
```
Comment (pernielsentikaer) [inline extensions/dawa/package.json:4]:
```suggestion
  "title": "DAWA - Danish Address Web API",
```

File: extensions/dawa/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "Data",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "icon": "dawa_icon.png",
+      "title": "Search Address",
+      "subtitle": "Search Denmarks addresses by adgangsadresseid",
```
Comment (pernielsentikaer) [inline extensions/dawa/package.json:18]:
```suggestion
      "subtitle": "DAWA",
```

File: extensions/dawa/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  * **Adgangsadresseid**: ${data?.id ?? "N/A"}
+  * **Adressebetegnelse**: ${data?.adressebetegnelse ?? "N/A"}
+  * **Kvhx**: ${data?.kvh ?? "N/A"}
+  * **Vejnavn:** ${data?.vejstykke.navn ?? "N/A"}
+  * **Husnr:** ${data?.husnr ?? "N/A"}
+  * **Etage:** ${data?.etage ?? "N/A"}
+  * **Dør:** ${data?.dør ?? "N/A"}
+  * **Supplerende bynavn:** ${data?.supplerendebynavn ?? "N/A"}
+  * **Postnr nummer:** ${data?.postnummer.nr ?? "N/A"}
+  * **Postnr navn:** ${data?.postnummer.navn ?? "N/A"}
```
Comment (pernielsentikaer) [inline extensions/dawa/src/index.tsx:47]: Keys should be in US English, data can be what the API returns

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* Seems like your metadata images are not made on a retina display, could you make it on your MacBooks display? If it's already done I might be able to help
* Can we figure out an English word for `adgangsadresseid`

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by SimonJ** [comment]
@pernielsentikaer Thanks for review Per 🙏 
Still hard with the **adgangsadresseid** I changed it in the markdown output. but it should still be in the "key" option so the user can see it's that key the extension wants to have, right?

**Pull request merged**